server.js

```diff
@@ -2,37 +2,128 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import booksData from './data/books.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+
+const Book = mongoose.model('Book', {
+  bookID: {
+    type: Number
+  },
+  title: {
+    type: String
+  },
+  authors: {
+    type: String
+  },
+  average_rating: {
+    type: Number
+  },
+  isbn: {
+    // unique: true,
+    type: String
+  },
+  isbn13: {
+    type: Number
+  },
+  language_code: {
+    type: String
+  },
+  num_pages: {
+    type: Number
+  },
+  ratings_count: {
+    type: Number
+  },
+  text_reviews_count: {
+    type: Number
+  },
+})
+
+if (process.env.RESET_DB) {
```

Comment:
Do be able to seed your deployed database you need to add this to heroku. could be the issue with not getting any data in atlas?